config/settings.yml

```diff
@@ -0,0 +1,8 @@
+plugins:
+  debtcollective_theme_enabled:
+    default: true
+    client: true
+  debtcollective_header_links:
+    default: "Dispute Your Debt, Dispute Your Debt, https://tools.debtcollective.org, vdo, self, remove"
```

Comment:
This should go to `/dispute-tools` like the other header links for "Dispute Your Debt" do, right? Or should we update the other ones to go just to the root tools?

Reply:
This will be set when we deploy Discourse, I removed the other since this is not meant to be used as safe defaults.